presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -237,11 +265,11 @@ public synchronized void load()
             lastRefresh.set(System.nanoTime());
         }
         catch (Throwable e) {
-            if (succinctNanos(System.nanoTime() - lastRefresh.get()).compareTo(maxRefreshInterval) > 0) {
-                lastRefresh.set(0);
-            }
             refreshFailures.update(1);
             log.error(e, "Error loading configuration from db");
+            if (lastRefresh.get() != 0) {
+                log.debug("Last successful configuration loading was %s ago", succinctNanos(System.nanoTime() - lastRefresh.get()).toString());
```

Comment:
Will this line be useful in test clusters?

Reply:
@highker : "Test clusters"? -- I am thinking it will help when something goes wrong and oncall can check the log.